lib/iso9660/iso9660_fs.c

```diff
@@ -1381,7 +1381,7 @@ typedef iso9660_stat_t * (stat_traverse_t)
   Get file status for psz_path into stat. NULL is returned on error.
   pathname version numbers in the ISO 9660
   name are dropped, i.e. ;1 is removed and if level 1 ISO-9660 names
-  are lowercased.
+  are downcased.
```

Comment:
I don't see that much difference between lowercased and downcased.  But whatever.

Reply:
Downcase is exclusively a verb (which is the intended case here), lowercase can be adjective or verb and is more commonly used as an adjective. Both would work, but while checking this uncommon usage I incorrectly concluded that lowercased was exclusively an adjective.